wkcuber/wkcuber/export_wkw_as_nifti.py

```diff
@@ -0,0 +1,206 @@
+import logging
+from argparse import ArgumentParser, Namespace
+from pathlib import Path
+from typing import Dict, List, Optional
+
+import nibabel as nib
+import numpy as np
+
+from wkcuber.api.bounding_box import BoundingBox
+from wkcuber.api.dataset import Dataset, LayerCategories
+from wkcuber.mag import Mag
+from wkcuber.utils import (
+    add_distribution_flags,
+    add_verbose_flag,
+    parse_bounding_box,
+    setup_logging,
+)
+
+
+def create_parser() -> ArgumentParser:
+    parser = ArgumentParser()
+
+    parser.add_argument(
+        "--source_path", "-s", help="Directory containing the wkw file.", required=True
+    )
+
+    parser.add_argument(
+        "--destination_path",
+        "-d",
+        help="Output directory for the generated nifti files.",
+        required=True,
+    )
+
+    parser.add_argument("--name", "-n", help="Name of the nifti", default="")
+
+    parser.add_argument(
+        "--bbox",
+        help="The bounding box of which the nifti file should be generated."
+        "The input format is x,y,z,width,height,depth."
+        "(By default, data for the full bounding box of the dataset is generated)",
+        default=None,
+        type=parse_bounding_box,
+    )
+
+    parser.add_argument(
+        "--mag", "-m", help="The magnification that should be read", default=1
+    )
+
+    parser.add_argument(
+        "--downsample", help="Downsample each nifti image", default=1, type=int
+    )
+
+    parser.add_argument(
+        "--original_bbox",
```

Comment:
```suggestion
        "--padded_bbox",
```

would fit better imo, if I understand its influence correctly.